src/Util.cc

```diff
@@ -209,6 +210,109 @@ std::string asFullPath(const std::string &_uri, const std::string &_filePath)
   // Use platform-specific separator
   return common::joinPaths(path,  uri);
 }
+
+//////////////////////////////////////////////////
+std::vector<std::string> resourcePaths()
+{
+  // Environment variable holding resource paths
+  const std::string kResourcePathEnv{"IGN_GAZEBO_RESOURCE_PATH"};
+
+  std::vector<std::string> gzPaths;
+  char *gzPathCStr = getenv(kResourcePathEnv.c_str());
+  if (gzPathCStr && *gzPathCStr != '\0')
+  {
+    gzPaths = common::Split(gzPathCStr, ':');
+  }
+
+  gzPaths.erase(std::remove_if(gzPaths.begin(), gzPaths.end(),
+      [](const std::string &_path)
+      {
+        return _path.empty();
+      }), gzPaths.end());
+
+  return gzPaths;
+}
+
+//////////////////////////////////////////////////
+void addResourcePaths(const std::vector<std::string> &_paths)
+{
+  // Environment variable holding resource paths
+  const std::string kResourcePathEnv{"IGN_GAZEBO_RESOURCE_PATH"};
```

Comment:
Is it possible to define the `IGN_GAZEBO_RESOURCE_PATH` and `SDF_PATH`  strings once?

Reply:
For this file, I moved them to local global variables: 0fe84c4f6680296f2099166d2de43890fa295bc9

Were you thinking of something else? Maybe defining them on the header and using throughout the code?

Reply:
Yeah, I was thinking of single place. That way the variable could be documented once, and we could eventually have a list of environment variables in a single place. I'm fine saving that for later.

Reply:
Yeah makes sense, moved it to a header: 7e679e960349970cc1af6ab3ba75c2d2dff269bc